frontend/src/global/constants.ts

```diff
@@ -69,6 +69,8 @@ export const ROUTES = {
     NEW_HOME: '/newconsole',
     NEW_LOGOS: '/newconsole/theming/logos',
     NEW_LOGOS_SIDEBAR: '/newconsole/theming/logos/sidebar',
+    NEW_THEME_FOOTER: '/newconsole/theming/footer',
+    NEW_THEME_FOOTER_SIDEBAR: '/newconsole/theming/footer/sidebar',
```

Comment:
@giantas, You don't need `NEW_THEME_FOOTER`, remove that. Set `NEW_THEME_FOOTER_SIDEBAR` to `/newconsole/theming/footer`.